apa-annotated-bibliography.csl

```diff
@@ -1,38 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" page-range-format="expanded">
   <info>
-    <title>American Psychological Association 6th edition (annotated bibliography with abstract)</title>
-    <title-short>APA</title-short>
+    <title>American Psychological Association 7th edition (annotated bibliography)</title>
+    <title-short>APA (annotated bibliography)</title-short>
```

Comment:
@bwiernik, did you drop the abstracts on purpose (and if so, why?). See the question at https://forums.zotero.org/discussion/80714/apa-7th-ed-with-annotations.

Reply:
It really feels like an error that abstract was ever used, rather than `note`/Extra. I think encouraging users to delete proper bibliographic data to store their annotations in Abstract is bad practice. We could revert it, but I would rather take the opportunity to encourage better bibliographic data management. @adam3smith

Reply:
Another option would be to add another variant, apa-annotated-bibliography-with-abstract.csl, that prints both abstract and note.